src/api/1inch/index.ts

```diff
@@ -1,11 +1,11 @@
 import { OrderKind } from '@cowprotocol/contracts'
-
 import { SupportedChainId as ChainId } from '@cowprotocol/cow-sdk'
-import { getTokensFromMarket } from 'legacy/utils/misc'
-import { getValidParams } from 'legacy/utils/price'
+import { PriceInformation } from 'types'
+
 import { LegacyPriceQuoteParams } from 'api/gnosisProtocol/legacy/types'
 import { fetchWithRateLimit } from 'common/utils/fetch'
-import { PriceInformation } from 'types'
+import { getTokensFromMarket } from 'legacy/utils/misc'
```

Comment:
I would treat legacy almost as code is not ours does it make sense to have it after "external modules"

Reply:
100% agree. I thought about it, but wasn't sure